chemprop/args.py

```diff
@@ -230,9 +267,22 @@ class TrainArgs(CommonArgs):
     """
     ignore_columns: List[str] = None
     """Name of the columns to ignore when :code:`target_columns` is not provided."""
-    dataset_type: Literal['regression', 'classification', 'multiclass', 'spectra']
+    dataset_type: Literal["regression", "classification", "multiclass", "spectra"]
     """Type of dataset. This determines the default loss function used during training."""
-    loss_function: Literal['mse', 'bounded_mse', 'binary_cross_entropy', 'cross_entropy', 'mcc', 'sid', 'wasserstein', 'mve', 'evidential', 'dirichlet'] = None
+    loss_function: Literal[
+        "mse",
+        "bounded_mse",
+        "binary_cross_entropy",
+        "cross_entropy",
+        "mcc",
+        "sid",
+        "wasserstein",
+        "mve",
+        "evidential",
+        "dirichlet",
+        "quantile",
```

Comment:
Is the inclusion of "quantile" here an accident? "quantile" doesn't seem to be supported in get_loss_func, so I'm guessing it's an accidental duplication of "quantile_interval".

Reply:
You are right, I originally implemented quantile loss but it has since been deleted.